core/modules/node/node.block.inc

```diff
@@ -80,6 +80,7 @@ class NodeBlock extends Block {
     }
 
     // Add 'submitted by' settings.
+    $content['#display_submitted'] = FALSE;
```

Comment:
Seems to me that something like `$content['#display_submitted'] = !empty($this->settings['display_submitted'];` here would be better here instead of this entire block of code.